checkov/secrets/plugins/custom_regex_detector.py

```diff
@@ -142,6 +142,9 @@ def _find_potential_secret(
         current_regex_to_metadata: dict[str, dict[str, Any]] = self.multiline_regex_to_metadata if is_multiline else self.regex_to_metadata
         kwargs["regex_denylist"] = current_denylist
         for match, regex in self.analyze_string(string_to_analyze, **kwargs):
+            if len(match) == 0:
+                # Skip empty matches
+                continue
```

Comment:
Isn't it a generator?

Reply:
yes it is

Reply:
a generator that returns string & regex
the value of match is str